packages/webpack/test/e2e-scuttle.spec.js

```diff
@@ -68,22 +76,53 @@ test(`webpack/scuttled - hosting globalThis's "Function" is scuttled`, async (t)
 })
 
 test(`webpack/scuttled - hosting globalThis's "Function" is scuttled excepted`, async (t) => {
-  await scuttle(t, {enabled: true, exceptions: ['Function']})
+  await scuttle(t, { enabled: true, exceptions: ['Function'] })
   t.notThrows(() => {
     t.is(new t.context.globalThis.Function('return 1')(), 1)
   }, 'Unexpected error in scenario')
 })
 
+test(`webpack/scuttled - webpackChunk global is transparently added to exceptions`, async (t) => {
+  await scuttle(t, { enabled: true, exceptions: ['Function'] })
+
+  t.notThrows(() => {
+    t.context.globalThis.webpackChunkTEST
+  }, 'Unexpected error in scenario')
+  t.truthy(t.context.globalThis.webpackChunkTEST)
+})
+
+test(`webpack/scuttled - webpackChunk global is transparently added to exceptions even when not specified 1`, async (t) => {
+  await scuttle(t, { enabled: true })
+
+  t.notThrows(() => {
+    t.context.globalThis.webpackChunkTEST
+  }, 'Unexpected error in scenario')
+  t.truthy(t.context.globalThis.webpackChunkTEST)
+})
+
+test(`webpack/scuttled - webpackChunk global is transparently added to exceptions even when not specified 2`, async (t) => {
+  await scuttle(t, true)
+
+  t.notThrows(() => {
+    t.context.globalThis.webpackChunkTEST
+  }, 'Unexpected error in scenario')
+  t.truthy(t.context.globalThis.webpackChunkTEST)
+})
+
 test(`webpack/scuttled - provided scuttlerName successfully invoked defined scuttlerFunc`, async (t) => {
-  const scuttlerName = 'SCUTTLER';
-  await scuttle(t, {
-    enabled: true,
-    scuttlerName,
-  }, {
-    [scuttlerName]: (globalRef, scuttle) => {
-      t.context.scuttler_func_called = true
-      scuttle(globalRef)
+  const scuttlerName = 'SCUTTLER'
+  await scuttle(
+    t,
+    {
+      enabled: true,
+      scuttlerName,
+    },
+    {
+      [scuttlerName]: (globalRef, scuttle) => {
+        t.context.scuttler_func_called = true
+        scuttle(globalRef)
```

Comment:
does this line serve a purpose?

Reply:
it demonstrates how scuttling integrates with Snow.